### DISCUSSION TITLE
How to set the skywalking UI query to only have a 10-second limit

### DISCUSSION THREAD

**Discussion by 983183947** [apache/skywalking#11268] [Q&A] [answered]
How to set the skywalking UI query to only have a 10-second limit

**Comment by weixiang1862** [comment]
SkyWalking works on minute level metrics, what do you mean `10-second limit`?

**Reply by 983183947** [reply]
<img width="1918" alt="image" src="https://github.com/apache/skywalking/assets/25240344/314a8531-b013-42ba-9ec8-bbcbe4edf3fa">

**Reply by weixiang1862** [reply, marked as answer]
As far as I konw, no timeout options can be configure now. You can modify `WebClient` client option yourself.
But I think fisrt priority thing is improving cluster perf, just increase timeout times will make things even worse.

![image](https://github.com/apache/skywalking/assets/16346644/f601b0e4-8d44-4b67-9b10-bcf3bd7a3e6e)
